README.md

```diff
@@ -34,27 +34,25 @@ git clone https://github.com/georgia-tech-db/eva.git
 2. Install the dependencies.
 ```shell
 sh script/install/before_install.sh
```

Comment:
Are these installation commands up to date?

Reply:
This installation script works for me, though I need to manually configure the mysql user and its privilege.